controllers/openshift_controller.go

```diff
@@ -686,12 +686,14 @@ func (r *KataConfigOpenShiftReconciler) createRuntimeClass() error {
 
 	foundRc := &nodeapi.RuntimeClass{}
 	err := r.Client.Get(context.TODO(), types.NamespacedName{Name: rc.Name}, foundRc)
-	if err != nil && k8serrors.IsNotFound(err) {
-		r.Log.Info("Creating a new RuntimeClass", "rc.Name", rc.Name)
-		err = r.Client.Create(context.TODO(), rc)
-		if err != nil {
-			return err
-		}
+	if err != nil && ! k8serrors.IsNotFound(err) {
+		return err
+	}
+
+	r.Log.Info("Creating a new RuntimeClass", "rc.Name", rc.Name)
+	err = r.Client.Create(context.TODO(), rc)
+	if err != nil {
+		return err
```

Comment:
@gkurz Won't this however create a new SCC even if one was found?

Reply:
oops of course...

Reply:
Then the other commit might have the same problem I suspect! :-)

Reply:
Exactcly :-D I've fixed both and repushed.